sktalk/corpus/conversation.py

```diff
@@ -98,6 +103,23 @@ def asdict(self):
         """
         return self._metadata | {"Utterances": [u.asdict() for u in self._utterances]}
 
+    @property
+    def metadata_df(self):
+        """Return the conversation metadata as a pandas dataframe."""
+        if not hasattr(self, "_metadata_df"):
```

Comment:
In such cases, I prefer to set the respective attribute to `None` in `__init__()`:
```python
self._metadata_df = None
```

Consequently, this check can be replaced:
```suggestion
        if self._metadata_df is None:
```

I find this a bit more explicit.